### ISSUE TITLE
Add a page to view the action log

### ISSUE THREAD

**Issue by czlee** [TabbycatDebate/tabbycat#46]
Maybe a live feed on the dashboard?

**Comment by czlee** [comment]
To avoid unnecessary work, should do this after #229

**Comment by czlee** [comment]
Does this need to be done soon enough for it to be in Chartreux? It depends on #229, which is a major refactor.

**Comment by philipbelesky** [comment]
Not really. I had added it before I dug back into how the log works doing the new homepage feeds. Agree that it needs a more abstracted solution if an overview page is going to make much sense.
